config/upstart.example/tfb.conf

```diff
@@ -0,0 +1,18 @@
+# /etc/init/tfb.conf
+env TFB_REPOPARENT="/private"
+env TFB_REPONAME="FrameworkBenchmarks"
+env TFB_REPOURI="git@github.com:ashawnbandy-te-tfb/FrameworkBenchmarks.git"
+env TFB_MAILINGLIST="sbandy@techempower.com"
```

Comment:
This needs to be split into 2 variables, one for the "To" field and one for the "From" field. Otherwise, if you try to send the results to multiple people, you'll end up with multiple email addresses in the "From" field. I don't know how email servers generally handle this, but it seems to me that, for the sake of email conventions, we should only be sending the email from a single address.